collect_app/src/androidTest/java/org/odk/collect/android/feature/formmanagement/DeleteBlankFormTest.java

```diff
@@ -89,4 +92,25 @@ public void afterFillingAForm_andDeletingIt_allowsFormToBeReDownloaded() {
                 .clickOK(new MainMenuPage(rule))
                 .startBlankForm("One Question");
     }
+
+    @Test
+    public void afterFillingAForm_andDeletingIt_allowsFormToBeReloadedDirectly() {
```

Comment:
It would be nice to have this tested at a lower level but the disk sync code is definitely not ready for that. For another day.